.github/workflows/binary-compatibility-checks.yml

```diff
@@ -23,13 +23,13 @@ jobs:
         scalaVersion: [ "2.12", "2.13" ]
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
           fetch-depth: 0
 
       - name: Set up JDK 8
-        uses: olafurpg/setup-scala@v10
+        uses: olafurpg/setup-scala@v13
```

Comment:
use coursier one?

Reply:
I fixed the build issues using https://github.com/apache/incubator-pekko/pull/53/commits/ca3e39807208d10f4c085802eb8db57e11dc85a0

Looks like sbtx command is now needed to get `-sbt-opts` to work - https://github.com/olafurpg/setup-scala#setup-scala-github-action

Reply:
@He-Pin would you be able to approve these changes if they are ok?

Reply:
So I am a bit confused here wrt to the usage of `sbtx`. Afaik `sbtx` shouldn't be necessary unless you have some ancient version of sbt and the standard `sbt` should work with sbt-opts fine.

Let me ping @dwijnand on this.

Reply:
these workflows use `sbt -sbt-opts` - this seems to only work with sbtx (sbt doesn't support this option)

Maybe older versions of olafurpg/setup-scala installed sbtx as if was sbt

I wouldn't have changed to sbtx if the workflows were working

Reply:
> I wouldn't have changed to sbtx if the workflows were working

Understood just want to understand whats going on (seems like `sbtx` is a workaround which is fine for now but we may need to migrate how we pass in sbt options later at some point so it just works with standard `sbt`).